<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QWidget" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>470</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_UserRegister">
     <property name="title">
      <string>User Registration</string>
     </property>
     <widget class="QLabel" name="label_Name">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>121</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Georgia</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Name and Surname</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_Email">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>100</y>
        <width>35</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Georgia</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Email</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_Birth">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>160</y>
        <width>77</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Georgia</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Date of Birth</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_Password">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>121</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Georgia</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_City">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>190</y>
        <width>25</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Georgia</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>City</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_Address">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>280</y>
        <width>51</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Georgia</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Address</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_Phone">
      <property name="geometry">
       <rect>
        <x>11</x>
        <y>70</y>
        <width>121</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Georgia</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Phone Number</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_Country">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>51</width>
        <height>24</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Georgia</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Country</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Register">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>60</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Register</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Clear">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>140</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_Back">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>220</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Back to Login</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Name">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>40</y>
        <width>311</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_PhoneNumber">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>70</y>
        <width>311</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Email">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>100</y>
        <width>311</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Password">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>130</y>
        <width>311</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Birth">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>160</y>
        <width>311</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_City">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>190</y>
        <width>311</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Country">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>220</y>
        <width>311</width>
        <height>24</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit_Address">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>260</y>
        <width>311</width>
        <height>101</height>
       </rect>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
